ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -30126,7 +29154,7 @@ periodics:
     repo: openshift-tests-private
   labels:
     ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws-autorelease-qe
+    ci-operator.openshift.io/cloud-cluster-profile: aws-sustaining-autorelease-412
```

Comment:
I think the change on this line comes from another PR.
Would you mind to rebase to latest code so that we have a cleaner git history ?